index.html

```diff
@@ -125,6 +125,40 @@ <h3 class="work-header">Multi-Post Stories</h3>
             </ul>
             <button type="button" class="work-card-button">See Project</button>
           </article>        
-        </section>  
+        </section>
+        <section class="contact-us">
+          <div class="contact-us-text">
+            <h2 class="contact-header">CONTACT US</h2>
+            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that
+              needs coding. I'd love to help with it!</p>
+          </div>
+          <form action="https://formspree.io/f/form_id" method="post" class="contact-form">
+            <ul class="contact-inputs">
+              <li>
```

Comment:
- Please set your formspree properly, on the section make use of the proper form_id unique to you, during registration on freespree.

Reply:
@Ginohmk Already done! Thank you :blush: , I wasn't sure about that part.